extensions/assessmentExtension/schema.json

```diff
@@ -1,78 +1,51 @@
 "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "Issuer Accreditation",
+  "title": "Assessment",
   "description": "This extension provides information about single or multiple assessments that would be completed by the recipient as part of the requirements for earning an OpenBadge. There could be multiple assessments of different types for each badge earned. Separate, independent evaluations of a single assessment could result in multiple assessment/evaluation records, all included in a single instance of the extension. ",
   "type": "object",
-  "definitions": {
-    "AssessmentArray": {
-      "description": "List of objects describing each assessment.",
-      "type": "array",
-      "items": {
-        "items": {
-        "$ref": "#/definitions/AssessmentObject"
-      }
+  "properties": {
+    "overallDescription": {
+      "description": " A description of how the assessment activity is organized, particularly describing the relationship between multiple assessments included in the extension.",
+      "type": "string"
     },
-    "AssessmentObject": {
```

Comment:
This change moves all of these properties outside of the `"assessment": []` array, but the example didn't change. Was that the intent of the change? It seems like this was correct the previous way.